bapi/bucketlist_models.py

```diff
@@ -0,0 +1,73 @@
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+
+from bapi import db, bcrypt
+from bapi.config import Config
+
+
+DB = db
```

Comment:
Check if you can import `db` directly from the init file without having to declare it again here